server/middleware/setUpCurrentCourtAppearance.ts

```diff
@@ -27,6 +29,11 @@ export default function setupCurrentCourtAppearance(
         res.locals.courtAppearanceCourtName = courtAppearance.courtCode
       }
     }
+
```

Comment:
One to discuss, but the appearance-outcomes look to be static ref data.. so would be potentially  more efficient to cache the appearance outcomes at some level rather than an api call every time